pylint/reporters/text.py

```diff
@@ -255,9 +255,35 @@ def handle_message(self, msg: Message) -> None:
         self.write_message(msg)
 
 
+class GithubReporter(TextReporter):
+    """Report messages in GitHub's special format to annotate code in its user
+    interface.
+    """
+
+    name = "github"
+    line_format = "::{category} file={path},line={line},endline={end_line},col={column},title={msg_id}::{msg}"
+    category_map = {
```

Comment:
We probably need to turn `MSG_TYPES_STATUS` into an `Enum`..

This is fine for now though!